src/OpenTK/Platform/MacOS/Cocoa/NSFloat.cs

```diff
@@ -179,5 +191,17 @@ struct NSRect
             return new RectangleF(s.Location, s.Size);
         } 
     }
-}
 
+    // Using IntPtr in NSFloat cause that if imported function
+    // return struct that consist of them you will get wrong data 
+    // This types are used for such function.
```

Comment:
Sequential is the default layout kind for structures. You are able to avoid some warnings about unused fields if you specify it directly, though.